Add documentation for SSH key deployment (#772)
red-hat-storage/ocs-ci: docs/getting_started.md

support deploying CI / Nightly builds. Please follow the instructions
[here](https://mojo.redhat.com/docs/DOC-1204026) to do so.

### SSH key

> We would like to use shared ssh key with engineering which allow us to connect

Reviewer: ```suggestion
We would like to use a shared ssh key with engineering which allows us to connect
```

to the nodes via known ssh key for QE and engineering.

> To setup with shared public ssh key for your deployment you have to follow

Reviewer: ```suggestion
To setup the shared public ssh key for your deployment you have to follow
```

> those steps:

Reviewer: ```suggestion
these steps:
```

Download private libra ssh key from secret location.

```console
$ wget https://secret.url.of.our.key -O ~/.ssh/libra.pem
chmod 600 ~/.ssh/libra.pem
ssh-keygen -y -f ~/.ssh/libra.pem > ~/.ssh/libra.pub
```

> Ask ppl on ocs-qe mailing list or chat room if you don't know where to find the

Reviewer: ```suggestion
Ask people on ocs-qe mailing list or chat room if you don't know where to find the
```

secret URL for libra key. Or look for this mail thread:
`SSH key deployed on our nodes` where the URL was mentioned.

> If you would like to use different path, you can overwrite it in custom

Reviewer: ```suggestion
If you would like to use a different path, you can overwrite it in the custom
```

config file under the DEPLOYMENT section with this key and value:
`ssh_key: "~/your/custom/path/ssh-key.pub"`.

> If you don't want to use shared key, you can change this value to

Reviewer: ```suggestion
If you don't want to use the shared key, you can change this value to
```
